docs/rl-algorithms/ppo_dna.md

```diff
@@ -0,0 +1,101 @@
+# Proximal Policy Gradient with Dual Network Architecture (PPO-DNA)
+
+## Overview
+
+PPO-DNA is a more sample efficient variant of PPO, based on using separate optimizers and hyperparameters for the actor (policy) and critic (value) networks.
+
+Original paper: 
+
+* [DNA: Proximal Policy Optimization with a Dual Network Architecture](https://arxiv.org/abs/2206.10027)
+
+## Implemented Variants
+
+
+| Variants Implemented      | Description |
+| ----------- | ----------- |
+| :material-github: [`ppo_dna_atari_envpool.py`](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_dna_atari_envpool.py), :material-file-document: [docs](/rl-algorithms/ppo_dna/#ppo_dna_atari_envpoolpy) | Uses the blazing fast Envpool Atari vectorized environment. |
+
+Below are our single-file implementations of PPO-DNA:
+
+## `ppo_dna_atari_envpool.py`
+
+The [ppo_dna_atari_envpool.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_dna_atari_envpool.py) has the following features:
+
+* Uses the blazing fast [Envpool](https://github.com/sail-sg/envpool) vectorized environment.
+* For Atari games. It uses convolutional layers and common atari-based pre-processing techniques.
+* Works with the Atari's pixel `Box` observation space of shape `(210, 160, 3)`
+* Works with the `Discrete` action space
+
+???+ warning
+
+    Note that `ppo_dna_atari_envpool.py` does not work in Windows :fontawesome-brands-windows: and MacOs :fontawesome-brands-apple:. See envpool's built wheels here: [https://pypi.org/project/envpool/#files](https://pypi.org/project/envpool/#files)
+
+
+### Usage
+
+```bash
+poetry install -E envpool
+python cleanrl/ppo_dna_atari_envpool.py --help
+python cleanrl/ppo_dna_atari_envpool.py --env-id Breakout-v5
+```
+
+### Explanation of the logged metrics
+
+See [related docs](/rl-algorithms/ppo/#explanation-of-the-logged-metrics) for `ppo.py`.
+
+### Implementation details
+
+[ppo_dna_atari_envpool.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_dna_atari_envpool.py) uses a customized `RecordEpisodeStatistics` to work with envpool but has the same other implementation details as `ppo_atari.py` (see [related docs](/rl-algorithms/ppo/#implementation-details_1)).
+
+Note that the original DNA implementation uses the `StickyAction` environment pre-processing wrapper (see (Machado et al., 2018)[^1]), but we did not implement it in [ppo_dna_atari_envpool.py](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_dna_atari_envpool.py) because envpool for now does not support `StickyAction`.
+
+
+### Experiment results
+
+Below are the average episodic returns for `ppo_dna_atari_envpool.py` compared to `ppo_atari_envpool.py`.
+
+
+| Environment      | `ppo_dna_atari_envpool.py` | `ppo_atari_envpool.py` |
+| ----------- | ----------- | ----------- | 
+| BattleZone-v5 (40M steps) | 74000 ± 15300 | 28700 ± 6300 |
```

Comment:
Do you mean 50M steps?

Reply:
Actually, my baseline `ppo_atari_envpool.py` experiments crashed before reaching full 50M steps, but I think the results are still valuable because they quite clearly show the performance difference between CleanRL baseline PPO and PPO-DNA.